### PR TITLE
Add Onboarding Step 5 - Industry

### LINKED ISSUE #1885: Onboarding - Profile Wizard: Step 5 - Industry
This step of the wizard should ask the user for the industry that they operate in. The industry should be saved in the onboarding data option via the REST API added in #1883.

The selection here will be used in other steps, so we should make sure to persist it somehow.

industry-1

industry-3

[Master Thread](https://wp.me/p90Yrv-10i) | [Designs](https://wp.me/p6riRB-4rG)

### PR DESCRIPTION
**Pull request by justinshreve**
Closes #1885.

This PR adds the industry step to the profile wizard.

Screen Shot 2019-05-28 at 12 57 51 PM

Screen Shot 2019-05-28 at 1 03 56 PM

To Test:
* Go to `/wp-admin/admin.php?page=wc-admin#/?step=industry` and confirm that you see the industries listed.
* Try selecting a couple, and hit save. You should be directed to the next page.
* Make a GET request to the profiler endpoint, and see that your industries are set.

### CODE REVIEW

File: includes/features/onboarding/class-wc-admin-onboarding.php [hunk 1]
```diff
@@ -51,13 +51,36 @@ public function should_show_profiler() {
 		return $is_completed || $is_skipped ? false : true;
 	}
 
+	/**
+	 * Get a list of allowed industries for the onboarding wizard.
+	 *
+	 * @return array
+	 */
+	public static function get_allowed_industries() {
```
Comment (joshuatf) [inline includes/features/onboarding/class-wc-admin-onboarding.php]: Nice 👍

File: client/dashboard/profile-wizard/style.scss [hunk 2]
```diff
@@ -171,6 +171,48 @@
...
+		.woocommerce-card__body {
+			padding-bottom: 0;
+		}
+
+		.woocommerce-profile-wizard__industry-actions {
+			padding-bottom: $gap;
+		}
+	}
+
+	.woocommerce-profile-wizard__continue {
```
Comment (joshuatf) [inline client/dashboard/profile-wizard/style.scss]: I've added some button styles for these buttons inside cards in https://github.com/woocommerce/woocommerce-admin/pull/2304 so we can probably remove this.

File: client/dashboard/profile-wizard/steps/industry.js [hunk 3]
```diff
@@ -2,52 +2,85 @@
...
+					{ __( 'In which industry does the store operate?', 'woocommerce-admin' ) }
+				</H>
+				<p>{ __( 'Choose any that apply' ) }</p>
+				<Card className="woocommerce-profile-wizard__industry-card">
+					<div className="woocommerce-profile-wizard__checkbox-group">
+						{ Object.keys( industries ).map( slug => {
+							return (
+								<CheckboxControl
+									key={ slug }
+									label={ __( industries[ slug ], 'woocommerce-admin' ) }
```
Comment (joshuatf) [inline client/dashboard/profile-wizard/steps/industry.js]: I think these are already localized in the API, so we might not need to localize these strings.

Comment (justinshreve) [reply client/dashboard/profile-wizard/steps/industry.js]: Good catch 👍. Fixed in https://github.com/woocommerce/woocommerce-admin/pull/2323/commits/45ec30ad54b2cb7da99735b94959a350714f5040.

File: client/dashboard/profile-wizard/style.scss (hunk 2 above)
Comment (justinshreve) [reply client/dashboard/profile-wizard/style.scss]: Removed 👍

File: client/dashboard/profile-wizard/steps/industry.js [hunk 4]
```diff
@@ -2,52 +2,85 @@
...
-			selectValue1: '1st',
-			selectValue2: '',
+			selected: [],
 		};
+		this.onContinue = this.onContinue.bind( this );
+		this.onChange = this.onChange.bind( this );
+	}
+
+	async onContinue() {
+		const updateProfile = await this.props.updateProfile( { industry: this.state.selected } );
```
[the remark this answers is not in the document]
Comment (justinshreve) [reply client/dashboard/profile-wizard/steps/industry.js]: Updated to `updateProfileItems` in https://github.com/woocommerce/woocommerce-admin/pull/2323/commits/45ec30ad54b2cb7da99735b94959a350714f5040.

**Comment by justinshreve** [comment]
Thanks for the review! I removed that CSS class, translation call, and made sure I'm using `updateProfileItems`.

**Pull request merged**